docker/integration/Dockerfile

```diff
@@ -0,0 +1,8 @@
+ARG BASE_IMAGE=exaworks/sdk-base
+FROM ${BASE_IMAGE}
+
+# install pytest-tap to allow Parsl's pytest scripts to output TAP
+RUN python3 -m pip install pytest-tap
+
+COPY test.sh /
+CMD /test.sh
```

Comment:
I would propose to use a different name (with the current name it will overwrite Parsl's individual test script) and maybe to put it in a dedicated directory, e.g., `/sdk_tests/` (later need to do the same with other test scripts)